internal/common/shared/forage.go

```diff
@@ -16,6 +16,9 @@ const (
 	FishForageType
 )
 
+// AllForageTypes specifies all available foraging types. Used for foraging history initialisation
+var AllForageTypes = []ForageType{DeerForageType, FishForageType}
```

Comment:
Ideally, this should be wrapped in a constant producing function (like gameConfig previously). But this is fine. Let's at least change it to a [slice literal](https://programming.guide/go/three-dots-ellipsis.html)
```suggestion
var AllForageTypes = [...]ForageType{DeerForageType, FishForageType}
```